conveyal/analysis-ui
# Scenario names in modifications

## Description

Show the scenario names next to the checkboxes to help users determine which scenario the modifications are active for.

Fixes #948 

## How to test

1. Go to a modification
2. Scroll to the bottom. Check and uncheck scenario boxes.
3. Add a new scenario. Ensure that the new scenario shows up in the modification.
